src/dropdown/dropdown.ts

```diff
@@ -117,7 +113,7 @@ export class NgbDropdown implements OnInit {
    * When it is 'outside' dropdowns are automatically closed on outside clicks but not on menu clicks.
    * When it is 'inside' dropdowns are automatically on menu clicks but not on outside clicks.
    */
-  @Input() autoClose: boolean | 'outside' | 'inside';
+  @Input() autoClose: AutoCloseType;
```

Comment:
`AutoCloseType` will be displayed as-is in the documentation pages. I would prefer to leave `boolean | 'inside' | 'outside'` just for readability reasons

Reply:
Too bad :( Documentation tools should be improved, but that's out of scope. It's theoretically a potential double maintenance for us then (changing one requires changing the other since they are linked).